org.civicrm.bootstrapcivicrm/scss/civicrm/common/_notifications.scss

```diff
@@ -59,3 +59,14 @@
     padding: 15px;
   }
 }
+
+#crm-main-content-wrapper {
+  form {
+    .messages {
+      &.crm-error {
+        position: absolute;
```

Comment:
visibility:hidden; hide the messge but keep the height in the page. After 2 seconds of activating the popup messages the height disappear. Without absolute, users will see the height of the page changes within 2 seconds of loading the page. 

If i replace visibility: hidden with display:none; => javascript popup library will not be able to detect the message in the page. so the solution is prevent such height changes using absolute position